src/components/conversation.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import { notFound } from "next/navigation";
+
+import { useChats } from "@/hooks/use-chats";
+
+import { Recommendation } from "./recommendation";
+
+export function Conversation({ uuid }: { uuid: string }) {
+  const { getChat, updateChat } = useChats();
+  const chat = getChat(uuid);
+
+  if (chat === null) {
+    notFound();
+  }
+
+  return (
+    <div className="flex max-h-[100dvh] flex-col overflow-y-auto">
```

Comment:
*Related to my comment on `src/app/chat/layout.tsx`*

**issue:** Remove those classes to implement proper scrolling behavior on this page
```suggestion
    <div className="flex flex-col">
```